### PR TITLE
ENG-284 replace code references on both component repository and page template

### PR DESCRIPTION
**Pull request by jeffgo10**
replaces all code references on the ff:
 - anything refers on digital exchange to component repository 
 - anything refers on page model to page template

API references such as API URI's and parameters/attributes are exempted in this change

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/component-repository/components/list/ComponentListPage.js [hunk 1]
```diff
@@ -2,12 +2,12 @@ import React from 'react';
...
-import ComponentListContainer from 'ui/digital-exchange/components/list/ComponentListContainer';
-import ComponentListViewModeSwitcherContainer from 'ui/digital-exchange/components/common/ComponentListViewModeSwitcherContainer';
+import SidebarContainer from 'ui/component-repository/SidebarContainer';
+import SearchBarContainer from 'ui/component-repository/components/SearchBarContainer';
+import FilterTypeContainer from 'ui/component-repository/components/FilterTypeContainer';
+import CategoryTabBarFilterContainer from 'ui/component-repository/CategoryTabBarFilterContainer';
+import ComponentListContainer from 'ui/component-repository/components/list/ComponentListContainer';
+import ComponentListViewModeSwitcherContainer from 'ui/component-repository/components/common/ComponentListViewModeSwitcherContainer';
 
 import { ROUTE_DE_CONFIG_LIST } from 'app-init/router';
```
Comment (ichalagashvili) [inline src/ui/component-repository/components/list/ComponentListPage.js:12]: hey, @jeffgo10 , I thinks this need to be changed to `ROUTE_ECR_CONFIG_LIST`

File: src/ui/component-repository/components/list/ComponentListPage.js [hunk 2]
```diff
@@ -17,8 +17,8 @@ const ComponentListPage = () => (
       <Row>
         <Col md={12}>
           <PageTitle
-            titleId="digitalExchange.component.list.title"
-            helpId="digitalExchange.component.help"
+            titleId="componentRepository.component.list.title"
+            helpId="componentRepository.component.help"
             configLink={ROUTE_DE_CONFIG_LIST}
```
Comment (ichalagashvili) [inline src/ui/component-repository/components/list/ComponentListPage.js:22]: here too

File: src/ui/component-repository/CategoryFilter.js [hunk 3]
```diff
@@ -2,24 +2,24 @@ import React from 'react';
...
 import { reduxForm } from 'redux-form';
-import CheckboxGroup from 'ui/digital-exchange/common/CheckboxGroup';
-import SidebarFilter from 'ui/digital-exchange/common/SidebarFilter';
+import CheckboxGroup from 'ui/component-repository/common/CheckboxGroup';
+import SidebarFilter from 'ui/component-repository/common/SidebarFilter';
 
-const CategoryFilterBody = ({ digitalExchangeCategories, intl, onChange }) => (
+const CategoryFilterBody = ({ componentRepositoryCategories, intl, onChange }) => (
   <SidebarFilter
     title={intl.formatMessage({ id: 'digitalExchange.sidebar.categoryFilterTitle' })}
```
Comment (ichalagashvili) [inline src/ui/component-repository/CategoryFilter.js:10]: I think this needs to be changed to: `componentRepository.sidebar.categoryFilterTitle`

File: src/ui/component-repository/components/list/ComponentList.js [hunk 4]
```diff
@@ -30,13 +30,15 @@ class ComponentList extends Component {
     this.props.fetchDEComponentsFiltered({ page: 1, pageSize });
```
Comment (ichalagashvili) [inline src/ui/component-repository/components/list/ComponentList.js:30]: I think this has to be changed to `fetchDEComponentsFiltered`

File: src/ui/component-repository/components/list/ComponentList.js [hunk 5]
```diff
@@ -81,7 +83,7 @@ ComponentList.propTypes = {
   fetchDEComponentsFiltered: PropTypes.func.isRequired,
```
Comment (ichalagashvili) [inline src/ui/component-repository/components/list/ComponentList.js:83]: same here: `fetchDEComponentsFiltered`

File: src/ui/component-repository/components/list/ComponentList.js (hunk 4 above)
Comment (ichalagashvili) [reply src/ui/component-repository/components/list/ComponentList.js:30]: on line 26 too needs to be changed to `fetchDEComponentsFiltered`

File: src/state/component-repository/components/reducer.js [hunk 6]
```diff
@@ -202,7 +206,7 @@ const filters = (state = filtersDefaultState, action = {}) => {
         ? addOrUpdateFilter(filter, state, category)
         : removeFilter(filter, state, category);
     }
-    case CLEAR_DE_SEARCH_FILTER: {
+    case CLEAR_ECR_SEARCH_FILTER: {
       const category = action.payload.digitalExchangeCategory;
```
Comment (ichalagashvili) [inline src/state/component-repository/components/reducer.js:210]: I think this needs to be changed to: `action.payload.componentRepositoryCategory`

**Comment by jeffgo10** [comment]
thanks @ichalagashvili. Man I still missed some of it. Please review my commits. thanks!

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/691#pullrequestreview-402418926" title="Approved">ichalagashvili</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [ichalagashvili]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**